lib/hybrid.h

```diff
@@ -460,6 +462,8 @@ struct HybridSimulator final {
 
       if (k != prev_k[i]) {
         if (k >= gatex.schmidt_decomp.size()) {
+          // Invalid path. Returns gatex index plus one to report error in case
```

Comment:
More detail here could be useful - e.g., why does this indicate an invalid path?